src/AddTodoForm.jsx

```diff
@@ -12,9 +20,16 @@ const AddTodoForm = ({ onAddTodo }) => {//onAddTodo prop, passed from App, to ad
     }
   };
 
+  const handleTitleChange = (event) => { 
+    event.preventDefault(); 
+    onAddTodo({title: todoTitle, id: Date.now()})
+    setTodoTitle(''); 
+  }
+
```

Comment:
Hi Nequil, it looks as if you may have overlooked these instructions in the homework:

![image](https://github.com/user-attachments/assets/e985b267-68cd-4fba-8d0e-bcb08e08364b)

Make sure you're getting the value from the input here and setting the state variable to that value whenever the user changes the value in the text input 👍 

Then in the `handleAddTodo` fn make sure you're referencing that state variable when calling the `onAddToDo` fn